scripts/dashboard/install-kcli-deps.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -ex
+
+sudo dnf -y install libvirt libvirt-daemon-driver-qemu qemu-kvm
+sudo usermod -aG qemu,libvirt $(id -un)
+newgrp libvirt
+systemctl enable --now libvirtd
+
+sudo dnf install -y podman
+
+mkdir -p ~/.ssh
+cp -R src/pybind/mgr/dashboard/ci/cephadm/.kcli ~
+
+echo "
+alias kcli='podman run --rm --net host --security-opt label=disable \
+    -v ${HOME}/.ssh:/root/.ssh \
+    -v ${HOME}/.kcli:/root/.kcli \
+    -v ${PWD}/src/pybind/mgr/dashboard/ci/cephadm:/workdir \
+    -v /var/lib/libvirt/images:/var/lib/libvirt/images \
+    -v /var/run/libvirt:/var/run/libvirt \
+    -v /var/tmp:/ignitiondir \
```

Comment:
I think you forget: 
`-v $PWD:/workdir `

Reply:
In fact is set to:
-v ${PWD}/src/pybind/mgr/dashboard/ci/cephadm:/workdir

Reply:
Updated anyway.